changelog

```diff
@@ -1,9 +1,11 @@
-turnkey-torrentserver-17.0 (1) turnkey; urgency=low
+turnkey-torrentserver-17.1 (1) turnkey; urgency=low
```

Comment:
This needs to remain as `v17.0` for now.

The resulting ISO will be v17.1 because the way we generate them this release. There is [historical context to that](https://www.turnkeylinux.org/blog/bug-v17.0-security-updates-kernel-panic).

We did it that way so we didn't need to update TKLDev and could just automatically rebuild the existing v17.0 appliances and keep building new ISO, even before [`fab` was fixed](https://github.com/turnkeylinux/fab/pull/17).

In a perfect world, we would have finished v17.0/v17.1 release long ago and have since released TKLDev v17.2 - with the fixed `fab` included.But unfortunately, that's not the world we live in...

FYI, the build process currently leverages [`bt-bugfix`](https://github.com/turnkeylinux/buildtasks/blob/master/bt-bugfix) to fix the bug in the resulting ISOs (built on TKLDev v17.1), increment the version (to v17.1) and add an automated changelog entry.

Reply:
Went ahead and changed this part in the changelog documentation to show the correct updated version.